Examples/Tests/scraper/PICMI_inputs.py

```diff
@@ -0,0 +1,209 @@
+##########################
+# numerics parameters
+##########################
+
+# --- Number of time steps
+max_steps = 10
+diagnostic_intervals = "::5"
+
+# --- Grid
+nx = 128
+ny = 16
+
+xmin = 0.0
+ymin = 0.0
+xmax = D_CA
+ymax = D_CA / nx * ny
+
+number_per_cell_each_dim = [32, 16]
+
+##########################
+# physics components
+##########################
+
+v_rms_elec = np.sqrt(constants.kb * T_ELEC / constants.m_e)
+v_rms_ion = np.sqrt(constants.kb * T_INERT / M_ION)
+
+uniform_plasma_elec = picmi.UniformDistribution(
+    density = PLASMA_DENSITY,
+    upper_bound = [None] * 3,
+    rms_velocity = [v_rms_elec] * 3,
+    directed_velocity = [0.] * 3
+)
+
+uniform_plasma_ion = picmi.UniformDistribution(
+    density = PLASMA_DENSITY,
+    upper_bound = [None] * 3,
+    rms_velocity = [v_rms_ion] * 3,
+    directed_velocity = [0.] * 3
+)
+
+electrons = picmi.Species(
+    particle_type='electron', name='electrons',
+    initial_distribution=uniform_plasma_elec
+)
+ions = picmi.Species(
+    particle_type='He', name='he_ions',
+    charge='q_e',
+    initial_distribution=uniform_plasma_ion
+)
+electrons.scrape_zmin = 1
+electrons.scrape_zmax = 1
+electrons.scrape_xmax = 1
+electrons.scrape_xmin = 1
+
+ions.scrape_zmin = 1
+ions.scrape_zmax = 1
+ions.scrape_xmax = 1
+ions.scrape_xmin = 1
+
+# MCC collisions
+cross_sec_direc = '../../../../warpx-data/MCC_cross_sections/He/'
+mcc_electrons = picmi.MCCCollisions(
+    name='coll_elec',
+    species=electrons,
+    background_density=N_INERT,
+    background_temperature=T_INERT,
+    background_mass=ions.mass,
+    scattering_processes={
+        'elastic' : {
+            'cross_section' : cross_sec_direc+'electron_scattering.dat'
+        },
+        'excitation1' : {
+            'cross_section': cross_sec_direc+'excitation_1.dat',
+            'energy' : 19.82
+        },
+        'excitation2' : {
+            'cross_section': cross_sec_direc+'excitation_2.dat',
+            'energy' : 20.61
+        },
+        'ionization' : {
+            'cross_section' : cross_sec_direc+'ionization.dat',
+            'energy' : 24.55,
+            'species' : ions
+        },
+    }
+)
+
+mcc_ions = picmi.MCCCollisions(
+    name='coll_ion',
+    species=ions,
+    background_density=N_INERT,
+    background_temperature=T_INERT,
+    scattering_processes={
+        'elastic' : {
+            'cross_section' : cross_sec_direc+'ion_scattering.dat'
+        },
+        'back' : {
+            'cross_section' : cross_sec_direc+'ion_back_scatter.dat'
+        },
+        # 'charge_exchange' : {
+        #    'cross_section' : cross_sec_direc+'charge_exchange.dat'
+        # }
+    }
+)
+
+##########################
+# numerics components
+##########################
+
+grid = picmi.Cartesian2DGrid(
+    number_of_cells = [nx, ny],
+    lower_bound = [xmin, ymin],
+    upper_bound = [xmax, ymax],
+    bc_xmin = 'dirichlet',
+    bc_xmax = 'dirichlet',
+    bc_ymin = 'periodic',
+    bc_ymax = 'periodic',
```

Comment:
Let's change this to 
```suggestion
    lower_boundary_conditions=['dirichlet', 'periodic'],
    upper_boundary_conditions=['dirichlet', 'periodic'],
```